lib/assertions.js

```diff
@@ -766,39 +766,65 @@ module.exports = (expect) => {
     );
   });
 
-  expect.addAssertion('<array-like> [not] to contain <any+>', function (
-    expect,
-    subject
-  ) {
-    const args = Array.prototype.slice.call(arguments, 2);
-    expect.withError(
-      () => {
-        args.forEach((arg) => {
-          expect(
-            subject &&
-              Array.prototype.some.call(subject, (item) =>
-                expect.equal(item, arg)
-              ),
-            '[not] to be truthy'
-          );
-        });
-      },
-      (e) => {
-        expect.fail({
-          diff:
-            expect.flags.not &&
-            ((output, diff, inspect, equal) =>
-              diff(
-                subject,
-                Array.prototype.filter.call(
+  expect.addAssertion(
+    [
+      '<array-like> to [only] contain <any+>',
+      '<array-like> [not] to contain <any+>',
+    ],
+    function (expect, subject) {
+      const args = Array.prototype.slice.call(arguments, 2);
+
+      expect.withError(
+        () => {
+          args.forEach((arg) => {
+            expect(
+              subject &&
```

Comment:
What is this truthiness check accomplishing? We know that the subject is array-like? (All the tests still pass if I remove it)

Reply:
I just carried an existing pattern forward :-) Might have been a left-over from before the type-system? I'll get rid of it.

Reply:
Removed :-)